CMakeLists.txt

```diff
@@ -241,6 +241,9 @@ option(POCO_ENABLE_STD_MUTEX "Set to OFF|NO using mutex from standard library (d
 if (POCO_ENABLE_STD_MUTEX)
 	add_definitions(-DPOCO_ENABLE_STD_MUTEX)
 endif ()
+
+option(POCO_USE_SENDFILE_FOR_HTTPSERVER "Set to OFF|ON using native sendFile function from OS for http server method HTTPServerResponse::sendFile" OFF)
```

Comment:
Let's try to avoid this kind of options if possible. We definitely can. We have couple option how this is not needed at all.

- We test at compile time if sendFile() is available
- We implement default behavior for sendFile() that it will always send file even when platform does not have sendfile operation.

I definetly would like option two. Does anyone know reason why we should not do that?

Reply:
I replaced option with cmake feature ```check_symbol_exists```